uw-frame-components/js/app-config.js

```diff
@@ -53,13 +53,9 @@ define(['angular'], function(angular) {
 
         })
         .value('FOOTER_URLS', [
-          { "url" : "/web/static/myuw-help",
+          { "url" : "https://my.wisc.edu/web/features",
```

Comment:
Would a relative path fare better than an absolute URL in encouraging frame-based app instances to integrate with the deployment tier they're deployed to?